docs/fundamentals/tools-and-productivity.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tools and productivity in .NET
+author: IEvangelist
+description: Learn about the various tooling, software development kits, diagnostic tools, and code analyzers available to .NET developers.
+ms.author: dapine
+ms.date: 10/20/2020
+ms.topic: overview
+---
+
+# Tools and productivity in .NET
+
+In this article, you'll learn about the various tools available to .NET developers. With .NET, you have a robust software development kit (SDK), that includes a command-line interface (CLI) namely, the .NET CLI. The .NET CLI powers many of the features throughout the .NET-ready integrated development environments (IDEs). This article also provides resources to productivity capabilities, such as .NET CLI tools for diagnosing performance issues, memory leaks, high CPU, deadlocks, and as well as tooling support for code analysis.
+
+## Tools
+
+Many tools are available to .NET developers, such as the .NET SDK, .NET CLI, IDEs, and so on.
```

Comment:
```suggestion
Many tools are available to .NET developers, such as the .NET SDK, .NET CLI, and IDEs.
```

Reply:
I would get rid of this h2 and elevate the h3s under it to h2s.